worker/worker.go

```diff
@@ -1412,26 +1411,28 @@ func isErrDuplicateTransactionSet(err error) bool {
 }
 
 func (w *worker) FundAccount(ctx context.Context, hk types.PublicKey, siamuxAddr string, balance types.Currency, revision *types.FileContractRevision) error {
-	return w.withHostV3(ctx, revision.ParentID, hk, siamuxAddr, func(h hostV3) error {
-		return h.FundAccount(ctx, balance, revision)
-	})
+	h, err := w.newHostV3(ctx, revision.ParentID, hk, siamuxAddr)
+	if err != nil {
+		return err
+	}
+	return h.FundAccount(ctx, balance, revision)
```

Comment:
Shouldn't we get rid of `FundAccount`, `Renew` and `SyncAccount` on the worker?
I think there's a couple of situations now where we'll do:

```
w.FundAccount()
if err w.SyncAccount()
w.FundAccount()
```
which will call `w.newHostV3` multiple times, instead we can just instantiate a host and call those methods directly?